getting_started/eeglab.md

```diff
@@ -9,34 +9,66 @@ tags: [dataformat, eeglab, eeg]
 
-## Using the EEGLAB DIPFIT plug-in
+## How does FieldTrip use EEGLAB?
 
-EEGLAB supports external plug-ins and one of them is DIPFIT. With the DIPFIT plugin, you can localize the sources of signal components that have been separated using ICA. The DIPFIT plugin is based on code from Robert Oostenveld, and the same code is included in FieldTrip.
+FieldTrip integrates EEGLAB functionalities to deal with component analysis. The function **[ft_componentanalysis](https://github.com/fieldtrip/fieldtrip/blob/release/ft_componentanalysis.m)** directly calls EEGLAB functions to perform the desired analysis as represented in the following figure:
 
-The old version 1.x of DIPFIT only fitted dipoles to EEG data using a 4-shell spherical head model using code that was specific to EEGLAB. The new version 2.x of DIPFIT is completely based on FieldTrip and also supports realistic BEM and FEM volume conduction models for EEG. Currently, EEGLAB cannot support some critical informations about the MEG sensors (the orientations of the gradiometers, mainly) so DIPFIT cannot be use with MEG signals.
+![How does FieldTrip use EEGLAB](/assets/img/getting_started/eeglab/FieldTrip_uses_EEGLAB.png)
+{% include image src="/assets/img/getting_started/eeglab/FieldTrip_uses_EEGLAB.png" width="300" %}
 
-To prevent overlapping programming efforts, the DIPFIT version 2.0 plugin uses the forward and inverse methods that are already implemented in FieldTrips **[ft_dipolefitting](https://github.com/fieldtrip/fieldtrip/blob/release/ft_dipolefitting.m)**) function. That means that you can use the DIPFIT graphical interface in EEGLAB, but you should have installed FieldTrip on your MATLAB path.
 
-## Using other FieldTrip functions
+## How does EEGLAB use FieldTrip?
 
-Besides using FieldTrip as the underlying "engine" for the DIPFIT plug-in of EEGLAB, it is also possible to export various aspects of your EEGLAB data structure completely to FieldTrip. Subsequently, you can continue your analysis of the data using the command-line FieldTrip functions
+EEGLAB integrates FieldTrip functions through the DIPFIT plug-in. This tool allows the localisation of the source signals from the components that have been separated using ICA. The main function used by DIPFIT is **[ft_dipolefitting](https://github.com/fieldtrip/fieldtrip/blob/release/ft_dipolefitting.m)**. It performs a grid search and non-linear fit with one or multiple dipoles and try to find the location where the dipole model is best able to explain the measured EEG topography, using realistic BEM and FEM volume conduction models.
```

Comment:
EEGLAB integrates FieldTrip functions through the DIPFIT and the FILE-IO plug-ins. 

<new paragraph> 

The DIPFIT plugin allows the localisation ... (including figure)

<new paragraph> 

The FILE-IO plugin allows reading raw data from disk into EEGLAB, from any of the file formats that is supported by FieldTrip. Most EEG file formats are directly supported in EEGLAB, but for some MEG and iEEG file formats this plugin is needed.

Reply:
_this is how I propose to change it_